wincred.go

```diff
@@ -4,10 +4,13 @@ package keyring
 
 import (
 	"strings"
+	"syscall"
 
 	"github.com/danieljoos/wincred"
 )
 
+const elementNotFoundError = syscall.Errno(1168)
```

Comment:
Cool, is there a reference we could link to for this?

Reply:
added